index.html

```diff
@@ -14,6 +15,116 @@
+          <h3 class="shopping-cart__title-header">Recently added items</h3>
+          <div class="shopping-cart__p-ammount">1 ITEM</div>
+        </div>
+
+        <ul class="shopping-cart__basket">
+          <li class="shopping-cart__item">
+            <div class="shopping-cart__item-content">
+              <figure class="shopping-cart__img">
+                <img src="media/product-1.jpg">
+              </figure>
+              <a href="/products/productId" class="shopping-cart__product">
+                <div class="shopping-cart__p-title">Vitamin C Plus Extended Release Family Size</div>
+                <div class="shopping-cart__p-qant">
+                  <div>
+                    <strong>Qty:</strong>
+                    <span class="shopping-cart__p-quant">1</span> x <span class="shopping-cart__p-price">£49.60</span>
+                  </div>
+                  <div class="shopping-cart__p-total">
+                    £<span>49.60</span>
+                  </div>
+                </div>
+              </a>
+            </div>
+            <a href="#" class="shopping-cart__remove" title="Remove product">
+              <svg class="svg-icon">
+                <use xlink:href="#icon-amway-icons-cancel"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="shopping-cart__item">
+            <div class="shopping-cart__item-content">
+              <figure class="shopping-cart__img">
+                <img src="media/product-1.jpg">
+              </figure>
+              <a href="/products/productId" class="shopping-cart__product">
+                <div class="shopping-cart__p-title">Vitamin C Plus Extended Release Family Size</div>
+                <div class="shopping-cart__p-desc">
+                  <strong>Color:</strong> Spice Bronze
+                </div>
+                <div class="shopping-cart__p-qant">
+                  <div>
+                    <strong>Qty:</strong>
+                    <span class="shopping-cart__p-quant">1</span> x <span class="shopping-cart__p-price">£49.60</span>
+                  </div>
+                  <div class="shopping-cart__p-total">
+                    £<span>49.60</span>
+                  </div>
+                </div>
+              </a>
+            </div>
+            <a href="#" class="shopping-cart__remove" title="Remove product">
+              <svg class="svg-icon">
+                <use xlink:href="#icon-amway-icons-cancel"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+
+      </div>
+
+      <div class="shopping-cart__total">
+        <div class="shopping-cart__subtotal">
+          <div>Cart subtotal:</div>
+          <div class="shopping-cart__subtotal-price">
+            <span>£</span><span class="shopping-cart__subtotal-item">49.60</span>
+          </div>
+        </div>
+
+        <div class="shopping-cart__btn">
+          <a href="#" class="header__cart-btn btn btn--secondary">
```

Comment:
Propose not to use dependence classes for -btn and other main elements
See .doc